.github/workflows/pr-check.yml

```diff
@@ -0,0 +1,21 @@
+name: Pull Request Check
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Java 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
```

Comment:
should we bump it up to 11?

Reply:
If we want to continue with the original approach of compiling with the minimum supported Dropwizard version, we should stick to Java 8.

https://github.com/ExpediaGroup/dropwizard-template-config/blob/ebcdceaad49bc4fae569ac330f632465c32aea38/pom.xml#L60-L63